helper_objects/barometer.py

```diff
@@ -14,6 +14,6 @@ def __init__(self):
         self.bmp.pressure_oversampling = 8
         self.bmp.temperature_oversampling = 2
 
-    # Returns the altitude in meters
+    # Returns the altitude in feet
     def get_altitude(self):
-        return self.bmp.altitude
\ No newline at end of file
+        return self.bmp.altitude*3.28084
```

Comment:
Nitpicky, but let's create a MT_TO_FT variable above and then use that in our return calculation for more readable code